lib/nostrum/struct/event/voice_state.ex

```diff
@@ -92,19 +92,19 @@ defmodule Nostrum.Struct.Event.VoiceState do
   @doc false
   def to_struct(map) do
```

Comment:
Thanks so much for testing the dev version and fixing this. It seems my recent changes here were quite the mess 🍅